### ISSUE TITLE
Optimize Android Brave Ads serving based upon local activity recognition

### ISSUE THREAD

**Issue by tmancey** [brave/brave-browser#11389]
See https://developers.google.com/location-context/activity-recognition

**Comment by tmancey** [comment]
Blocked as needs discussion

**Comment by tmancey** [comment]
Unblocked as discussed with @jsecretan and if no activity has occurred for 1 hour ads should not be shown

**Comment by tmancey** [comment]
Closing as a duplicate of https://github.com/brave/brave-browser/issues/17499
